deploy/charts/venafi-kubernetes-agent/templates/deployment.yaml

```diff
@@ -89,23 +105,6 @@ spec:
             {{- with .Values.volumeMounts }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
-          env:
-          - name: POD_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-          - name: POD_NAME
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.name
-          - name: POD_UID
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.uid
-          - name: POD_NODE
-            valueFrom:
-              fieldRef:
-                fieldPath: spec.nodeName
```

Comment:
Apologies. I introduced this bug, in https://github.com/jetstack/jetstack-secure/pull/589/commits/5679dc52aeab5d1ac54462fe3fd800407bf3e394